information-resources

```diff
@@ -0,0 +1,39 @@
+#Information resources
```

Comment:
Adding a `.md` extension to this file will help make this more readable on GitHub.